querysql/querysql.go

```diff
@@ -179,9 +184,17 @@ func Next(rs *ResultSets, scanner Target) error {
 	}()
 
 	if !rs.started {
-		if err := rs.processAllLogSelects(); err != nil {
+		noResultSets, err := rs.processAllLogSelects()
+		if err != nil {
 			return err
 		}
+		if noResultSets {
```

Comment:
This is inverted from the way I would normally have written it (hasResultSets). Would it be ok to change it to hasResultSets instead?

Reply:
Well, I don't really know that there is a result set.

I just didn't get the specific signal that there isn't a result set.

But I can name it `hadColumns` I guess

Reply:
Pushed; "noResultSets" was doing the interpretation in the wrong function, better to report on `hadColumns`